api/docs/release.dox

```diff
@@ -203,8 +203,9 @@ Further non-compatibility-affecting changes include:
    runtime option which locates guest system libraries and enables workarounds for
    problems with QEMU's threads.
  - Added dr_is_detaching(), an API to query whether detach is in progress.
- - Added instr_zeroes_zmmh() that returns true if an instruction clears the
```

Comment:
Bug: you've deleted this entry!

Reply:
Ah sorry! Sync issues from when I split off these changes from my old PR. Fixed it.